experimental/traffic-portal/src/app/core/core.module.ts

```diff
@@ -57,6 +58,7 @@ import { UsersComponent } from "./users/users.component";
 
 export const ROUTES: Routes = [
 	{ component: DashboardComponent, path: "" },
+	{ component: AsnsTableComponent, path: "asns" },
```

Comment:
There needs to be a route entry for the details page, too. It isn't currently accessible.

Reply:
Yes, that is correct because there is nothing for ASN details yet.

Reply:
I could swear this PR included a details page. But I'm not seeing one now, so... aight